### PR TITLE
docs: fix the docs url of installation instructions

### PR DESCRIPTION
**Pull request by haoxins**
## Which issue does this PR close?

docs only

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,8 +69,8 @@ Linux, Apple OSX (Intel and M1)
...
 ## Getting started
 
-See the [DataFusion Comet User Guide](https://datafusion.apache.org/comet/user-guide/) for installation instructions.
+See the [DataFusion Comet User Guide](https://datafusion.apache.org/comet/user-guide/installation.html) for installation instructions.
 
 ## Contributing
 See the [DataFusion Comet Contribution Guide](https://datafusion.apache.org/comet/contributor-guide/contributing.html)
-for information on how to get started contributing to the project.
\ No newline at end of file
+for information on how to get started contributing to the project.
```
Comment (viirya) [inline README.md:76]: What the difference on this sentence? Seems it is same but it is marked as in diffs.

**Review by viirya — APPROVED** [review]
Thanks @haoxins

File: README.md (hunk 1 above)
Comment (haoxins) [reply README.md:76]: I think it's a new line at the end of the file.
I edited this from the GitHub UI.

**Comment by viirya** [comment]
Thanks @haoxins

**Pull request merged**